### PR TITLE
Update creating-nuget-packages.md

### PR DESCRIPTION
**Pull request by JasonWHowell**
Shortening title as per SEO audit item 1656860
Currently the title truncates in Google:
image

### CODE REVIEW

File: docs/core/deploying/creating-nuget-packages.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Create a NuGet Package with .NET Core command-line interface (CLI) tools
+title: Create a NuGet Package with .NET Core command-line tools
```
Comment (mairaw) [inline docs/core/deploying/creating-nuget-packages.md]: We can actually just say .NET Core CLI 
https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=44109

```suggestion
title: Create a NuGet Package with .NET Core CLI
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @JasonWHowell. I've made a suggestion.

File: docs/core/deploying/creating-nuget-packages.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Create a NuGet Package with .NET Core command-line interface (CLI) tools
+title: Create a NuGet Package with .NET Core CLI
```
Comment (mairaw) [inline docs/core/deploying/creating-nuget-packages.md]: Sorry, just noticed something else that was wrong

```suggestion
title: Create a NuGet package with .NET Core CLI
```

**Review by mairaw — APPROVED** [review]
LGTM

**Comment by JasonWHowell** [comment]
Build looks good to me 
https://review.docs.microsoft.com/en-us/dotnet/core/deploying/creating-nuget-packages?branch=pr-en-us-16356

**Pull request merged**